public/locales/en/translation.json

```
@@ -0,0 +1,767 @@
+      "failCreateFeature": "Unable to create feature :",
+      "successDeleteFeature": "Feature has been successfully deleted.",
+      "failDeleteFeature": "Feature has not been deleted.",
+      "noAccess": "You are not allowed to access this resource.",
+      "geometry": "Geometry",
+      "true": "Yes",
+      "false": "No",
+      "other": "Other",
+      "generatedDocuments": "Generated documents",
+      "fitBoundButton": "Center map to \"{{name}}\"",
+      "menu": {
+        "default": "Informations",
+        "geometries": "Geometries",
+        "attachments": "Attachments"
+      },
+      "attachment": {
+        "allowedFileTypes": "Files ({{files}}) allowed",
+        "confirmDeletionText": "Are you sure you want to delete {{name}}? This can't be canceled.",
+        "delete": "Delete file",
+        "empty": "No file for this feature",
+        "figcaptionCancel" : "Cancel",
+        "figcaptionEdit" : "Edit legend",
+        "figcaptionSubmit" : "Submit legend",
+        "files": "Files",
+        "images": "Images",
+        "openImage": "Show image in new window",
+        "label": "Legend",
+        "loadFiles": "Load file(s)",
+        "noFiles": "No files",
+        "category" : {
+          "helper": "Files will be linked to this category",
+          "select": "Select category",
+          "input": "Set category",
+          "new": "+ New category"
+        }
+      }
+    }
+  },
+  "help": {
+    "block-heading": "Advices :"
+  },
+  "jsonSchema": {
+    "geometryField": {
+      "type": {
+        "Point": {
+          "singular": "a point"
+        },
+        "LineString": {
+          "singular": "a line"
+        },
+        "Polygon": {
+          "singular": "an area"
+        },
+        "MultiPoint": {
+          "singular": "some points"
+        },
+        "MultiLineString": {
+          "singular": "some lines"
+        },
+        "MultiPolygon": {
+          "singular": "some areas"
+        },
+        "GeometryCollection": {
+          "singular": "some geometries"
+        }
+      },
+      "title": "Coordinates (decimal WGS84)",
+      "latitude": "Latitude",
+      "longitude": "Longitude",
+      "helper-create": "Draw $t(jsonSchema.geometryField.type.{{type}}.singular) with available button on map.",
+      "helper-edit": "Edit $t(jsonSchema.geometryField.type.{{type}}.singular) with available button on map.",
+      "save": "Once edited, save by cubmit button."
+    },
+    "panelError": {
+      "title": "Your input has some errors"
+    },
+    "RTEField": {
+      "close": "Close",
+      "pick-image": "Select picture"
+    }
+  },
+  "user": {
+    "project": "Users",
+    "nav": {
+      "list": "User list"
+    },
+    "form": {
+      "id": "Id",
+      "email": "Email",
+      "uuid": "Uuid",
+      "superuser": "Super user",
+      "active": "Active",
+      "permissions": "Permissions",
+      "additional-information": "Additional informations",
```

```suggestion
      "additional-information": "Additional information",
```
I think information has no plural...